imap-types/src/datetime.rs

```diff
@@ -11,6 +11,7 @@ use serde::{Deserialize, Serialize};
 use crate::datetime::error::{DateTimeError, NaiveDateError};
 
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "serde", serde(try_from = "chrono::DateTime<FixedOffset>"))]
```

Comment:
I'm not sure I understand this. Is this to get rid of the newtype? Is serde handling this automatically with `try_from`?